<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Expert</class>
 <widget class="QMainWindow" name="Expert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>270</width>
      <height>102</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Averaging</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>140</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblAvgAlgo">
        <property name="text">
         <string>Average Algorithm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblWindowSize">
        <property name="text">
         <string>Window size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSmoothingFactor">
        <property name="text">
         <string>Smoothing Factor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>109</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QEComboBox" name="avgAlgo">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="variable" stdset="0">
         <string>SOFB:MovAvg:Algo</string>
        </property>
        <property name="writeOnChange">
         <bool>true</bool>
        </property>
        <property name="displayAlarmStateOption" stdset="0">
         <enum>QEComboBox::Always</enum>
        </property>
        <property name="useDbEnumerations">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QENumericEdit" name="windowSize">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="variable" stdset="0">
         <string>SOFB:MovAvg:WindowSize</string>
        </property>
        <property name="autoScale">
         <bool>false</bool>
        </property>
        <property name="leadingZeros">
         <number>4</number>
        </property>
        <property name="precision">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="writeOnEnter">
         <bool>false</bool>
        </property>
        <property name="writeOnFinish">
         <bool>false</bool>
        </property>
        <property name="writeOnChange">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QENumericEdit" name="smoothingFactor">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="variable" stdset="0">
         <string>SOFB:MovAvg:SmoothingFactor</string>
        </property>
        <property name="autoScale">
         <bool>false</bool>
        </property>
        <property name="leadingZeros">
         <number>1</number>
        </property>
        <property name="precision">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="writeOnEnter">
         <bool>false</bool>
        </property>
        <property name="writeOnFinish">
         <bool>false</bool>
        </property>
        <property name="writeOnChange">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="footer">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>200</y>
      <width>270</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>307</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblRfOnly">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Run RF Correction Only</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDebugMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>144</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Mode (no correction)</string>
    </property>
   </widget>
   <widget class="QECheckBox" name="chkBoxDebugMode">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>145</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="variable" stdset="0">
     <string>SOFB:NoSetPv</string>
    </property>
   </widget>
   <widget class="QECheckBox" name="chkBoxRfOnly">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>129</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="variable" stdset="0">
     <string>SOFB:OnlyRf</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QECheckBox</class>
   <extends>QCheckBox</extends>
   <header>QECheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>QEComboBox</class>
   <extends>QComboBox</extends>
   <header>QEComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
